src/ProjectTemplates/Web.ProjectTemplates/Microsoft.DotNet.Web.ProjectTemplates.csproj

```diff
@@ -23,6 +23,8 @@
       MicrosoftNETCoreAppRuntimeVersion=$(MicrosoftNETCoreAppRuntimeVersion);
       SystemNetHttpJsonPackageVersion=$(SystemNetHttpJsonPackageVersion);
       MicrosoftGraphPackageVersion=$(MicrosoftGraphPackageVersion);
+      MicrosoftAspNetCoreAuthenticationJwtBearerPackageVersion=$(MicrosoftAspNetCoreAuthenticationJwtBearerPackageVersion);
+      MicrosoftAspNetCoreAuthenticationOpenIdConnectPackageVersion=$(MicrosoftAspNetCoreAuthenticationOpenIdConnectPackageVersion);
```

Comment:
This means we _never_ test against the just-built version of these packages. I doubt that's intentional. Suggest making this conditional on `$(VersionSuffix)` being either `ci` or `dev`.

Conditionally add `@(PackageVersionVariableReference)` items below for the opposite case. That way non-local and non-PR builds will test the new stuff.